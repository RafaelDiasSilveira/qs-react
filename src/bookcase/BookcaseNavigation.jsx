import React from 'react';

export default () => (
  <div>
    <h3 className="text-center">Total de livros na estante: 0/10</h3>
    <button className="btn btn-primary btn-lg btn-block">
      Adicionar livro
    </button>
  </div>
);
