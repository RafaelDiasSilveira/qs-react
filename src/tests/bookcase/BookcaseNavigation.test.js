import React from 'react';
import { shallow } from 'enzyme';

import BookcaseNavigation from '../../bookcase/BookcaseNavigation';

test('should render BookcaseNavigation correctly', () => {
  const wrapper = shallow(<BookcaseNavigation />);

  expect(wrapper).toMatchSnapshot();
});
