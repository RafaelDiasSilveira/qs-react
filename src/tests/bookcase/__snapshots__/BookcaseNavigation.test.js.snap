// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BookcaseNavigation correctly 1`] = `
<div>
  <h3
    className="text-center"
  >
    Total de livros na estante: 0/10
  </h3>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="btn btn-primary btn-lg btn-block"
    to="/book/new"
  >
    Adicionar livro
  </NavLink>
</div>
`;
